src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -255,10 +255,10 @@ namespace winrt::TerminalApp::implementation
 
             std::wstring pathText = path.wstring();
 
-            // Handle edge case of "C:\\", seems like the "StartingDirectory" doesn't like path which ends with '\'
```

Comment:
I'm actually quite surprised by this premise. The reason we have issues with StartingDirectory when passed through the commandline are ones of _escaping_.

Does it really beef it if we don't do this trick at all? *huh.*

Reply:
(because the commandline escaping for `wt` resaults in `"C:\"` being parsed as `C:\"` (terminating quote captured in path))

Reply:
![image](https://user-images.githubusercontent.com/189190/127901534-9d4281b4-a02a-4a90-8914-22e75843abe4.png)

Reply:
Yes - this is purely for the "Open in new window" case

If I comment out the `pathText.push_back(L'.')` and drop on `C:\` with shift pressed, the new window fails to parse the command line:

![image](https://user-images.githubusercontent.com/5821575/127902800-df6eb665-66c3-44d0-8bac-d41b01160cbb.png)

![image](https://user-images.githubusercontent.com/5821575/127902810-ae5b6792-8ba1-42b8-984b-00f73610e5af.png)

Reply:
Drag and dropping `C:\` to get a new tab or split the current window works fine, it's just the new window case that fails.

Reply:
*OH*. Okay.

We should fix this in the New Window commandline generator, honestly. That would also fix it for shift-clicking the `cmd2` profile I created to test this. :smile:

I suspect that this code lives in `NewTerminalArgs::ToCommandline`.

Reply:
Done - with the added bonus that this now has a test 🙂

Reply:
Love it! Thank you!